package.json

```diff
@@ -36,6 +36,9 @@
         "url": "https://github.com/bitinn/node-fetch/issues"
     },
     "homepage": "https://github.com/bitinn/node-fetch",
+    "dependencies": {
+        "whatwg-url": "^5.0.0"
```

Comment:
why this version?
actual is 9.1.0

this breaks our code because we have dependency on whatwg-url v.6.5.0 already

Reply:
> why this version?
> actual is 9.1.0

This is the latest version that supports Node.js 4.x

> this breaks our code because we have dependency on whatwg-url v.6.5.0 already

In which way does your app break? 🤔